n64.mk

```diff
@@ -1,3 +1,15 @@
+ifdef SystemRoot # We are running on windows
+  FIXPATH =$(subst /,\,$1)
+  MKDIR = mkdir
+  MV = move
+  RM = del /F /Q # force deletion quietly
+else # We are running on UNIX
+  FIXPATH = $1
+  MKDIR = mkdir -p # make parent directories as needed
+  MV = mv -f # ignore nonexistent files, never prompt
+  RM = rm -f # ignore nonexistent files, never prompt
+endif
```

Comment:
I understand the effort here, but we have a maintenance problem. This means that whatever we do to n64.mk from now on, we need to have an implementation also for the very limited CMD shell in Windows. This concerns me.
Do you have any alternative solution in mind? Why don't you like the msys solution?